systemverilog-plugin/UhdmAst.cc

```diff
@@ -3713,17 +3713,31 @@ void UhdmAst::process_tagged_pattern()
     AST::AstNode *lhs_node = nullptr;
     if (assign_node) {
         assign_type = assign_node->type;
-        lhs_node = assign_node->children[0];
+        lhs_node = assign_node->children[0]->clone();
     } else {
         lhs_node = new AST::AstNode(AST::AST_IDENTIFIER);
         auto ancestor = find_ancestor({AST::AST_WIRE, AST::AST_MEMORY, AST::AST_PARAMETER, AST::AST_LOCALPARAM});
```

Comment:
While you're there: `const auto* const`.
Same above, in `auto assign_node = ...`. This change will actually make `->clone()` you've added mandatory.